watertap/examples/flowsheets/full_treatment_train/util.py

```diff
@@ -12,40 +12,28 @@
 ###############################################################################
 
 from pyomo.environ import check_optimal_termination, TransformationFactory
+from pyomo.util.check_units import assert_units_consistent
 from idaes.core.util import get_solver
 from idaes.core.util.model_statistics import degrees_of_freedom
 from idaes.core.util.scaling import (unscaled_variables_generator,
                                      unscaled_constraints_generator,
                                      calculate_scaling_factors)
-from pyomo.util.check_units import assert_units_consistent
+from watertap.core.util.initialization import assert_degrees_of_freedom, check_solve as _check_solve
 
-
-# NOTE: In a full flowsheet, you will want to leave the default values for
-#       bound_push and mu_init as is!!! However, if your flowsheet or block
-#       contains ONLY chemistry, then setting bound_push = 1e-10 and mu_init = 1e-6
-#       works ver well. 
-def solve_with_user_scaling(blk, solver=None, tee=False, fail_flag=True, bound_push=0.1, mu_init=1e-1):
+def solve_block(blk, solver=None, tee=False, fail_flag=True):
     if solver is None:
-        solver = get_solver(options={'nlp_scaling_method': 'user-scaling',
-                                     'bound_push': bound_push,
-                                     'mu_init': mu_init})
+        solver = get_solver(options={'tol':1e-11})
     results = solver.solve(blk, tee=tee)
     if fail_flag:
         check_solve(results)
 
 
 def check_dof(blk, dof_expected=0):
```

Comment:
Do we need this if we already of the same or similar method in `core/util/initialization` ? Guessing we're keeping this to avoid having to replace imports everywhere else.